maildir2gmail.py

```diff
@@ -56,12 +59,29 @@ def append(self, filename):
 
         message = email.message_from_string(content)
         timestamp = parsedate(message['date'])
-        if not timestamp:
-            log('Skipping "%s" - no date' % os.path.basename(filename))
+        fileTimestamp = os.path.getctime(filename)
+	if not self.ignore_missing_date:
```

Comment:
Am I confused or is the indentation not correct (missing one space)?
Not sure if Python complains about it though ...

Reply:
Oh, you're right. That's not correct. But for some reason it seems to work. I've tested all my changes under real conditions.

Reply:
Well, can you please align it, to make it look consistent?

Reply:
Done.